index.html

```diff
@@ -1331,7 +1342,7 @@ <h3>Issuer</h3>
 RECOMMENDED that dereferencing the <a>URI</a>, or the <code>id</code> property,
 results in a document containing machine-readable information about the
 <a>issuer</a> that can be used to <a>verify</a> the information expressed in
-the <a>credential</a>.
+the <a>verifiable credential</a>.
```

Comment:
This change is not correct, we meant "credential".

Reply:
Fixed.